app/src/main/java/org/mozilla/fenix/experiments/NimbusFeatures.kt

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.experiments
+
+import android.content.Context
+import android.os.StrictMode
+import org.mozilla.experiments.nimbus.mapKeysAsEnums
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.ext.getVariables
+
+class NimbusFeatures(
+    private val context: Context,
+) {
+
+    val homeScreen : HomeScreenFeatures by lazy  {
+        HomeScreenFeatures(context)
+    }
+
+    class HomeScreenFeatures(private val context: Context) {
+        /**
+         * `FeatureId.HOME_PAGE` feature; the complete JSON, is shown here:
+         *
+         * ```json
+         * {
+         *     "sections-enabled": {
+         *         "topSites": true,
+         *         "recentlySaved": false,
+         *         "jumpBackIn": false,
+         *         "pocket": false,
+         *         "recentExplorations": false
+         *     }
+         * }
+         * ```
+         */
+
+        /**
+         * This enum accompanies the `FeatureId.HOME_PAGE` feature.
+         *
+         * These names here should match the names of entries in the JSON.
+         */
+        @Suppress("EnumEntryName")
+        private enum class HomeScreenSection(val default: Boolean) {
+            topSites(true),
+            recentlySaved(false),
+            jumpBackIn(false),
+            pocket(false),
+            recentExplorations(false),
+        }
+
+        private val homeScreenFeatures: Map<HomeScreenSection, Boolean> by lazy {
+            val experiments =
+                context.components.strictMode.resetAfter(StrictMode.allowThreadDiskReads()) {
+                    context.components.analytics.experiments
+                }
+            val variables = experiments.getVariables(FeatureId.HOME_PAGE, false)
+            val sections: Map<HomeScreenSection, Boolean> =
+                variables.getBoolMap("sections-enabled")?.mapKeysAsEnums() ?: mapOf()
+            sections.withDefault { key -> key.default }
+        }
+
+        fun isRecentlyTabsActive(): Boolean {
+            return homeScreenFeatures[HomeScreenSection.jumpBackIn] == true
+        }
+
+        fun isRecentlySavedActive(): Boolean {
+            return homeScreenFeatures[HomeScreenSection.recentlySaved] == true
+        }
+
+        fun isRecentExplorationsActive(): Boolean {
+            return homeScreenFeatures[HomeScreenSection.recentExplorations] == true
+        }
+
+        fun isPocketRecommendationsActive(): Boolean {
+            return homeScreenFeatures[HomeScreenSection.pocket] == true
+        }
+
+        fun isTopSitesActive(): Boolean {
+            return homeScreenFeatures[HomeScreenSection.topSites] == true
+        }
```

Comment:
This needs to be wired up to the `HomeFragment`.

Reply:
Yes, I'm a bit confused about it, I'm not sure how to approach this one, as it is a stable feature, it should always show unless nimbus indicates otherwise. Should I wire it up and rely solely on what the nimbus variables indicate?